products/rhel10/profiles/ospp.profile

```diff
@@ -11,10 +11,11 @@ reference: https://www.niap-ccevs.org/Profile/Info.cfm?PPID=469&id=469
 title: 'DRAFT - Protection Profile for General Purpose Operating Systems'
 
 description: |-
-    This profile is part of Red Hat Enterprise Linux 9 Common Criteria Guidance
+    This is draft profile is not based on the Red Hat Enterprise Linux 10 riteria Guidance
```

Comment:
This draft profile is not based on the Red Hat Enterprise Linux 10 Common Criteria Guidance